sycl/source/detail/graph_impl.hpp

```diff
@@ -919,6 +919,10 @@ class graph_impl : public std::enable_shared_from_this<graph_impl> {
   /// @return Context associated with graph.
   sycl::context getContext() const { return MContext; }
 
+  const DeviceImplPtr &getDeviceImplPtr() const {
```

Comment:
I know this is trivial but it stands out the the other methods have a doxy comment and this one doesn't, could you add one for consistency.